server/poetry.lock

```diff
@@ -268,6 +268,103 @@ files = [
     {file = "packaging-24.1.tar.gz", hash = "sha256:026ed72c8ed3fcce5bf8950572258698927fd1dbda10a5e981cdf0ac37f4f002"},
 ]
 
+[[package]]
+name = "pillow"
+version = "10.4.0"
+description = "Python Imaging Library (Fork)"
+optional = false
+python-versions = ">=3.8"
+files = [
```

Comment:
Could you explain what this is and what it does

Reply:
![image](https://github.com/user-attachments/assets/d8b0374a-7bfb-491e-9b07-88037618a7b8)
In models.py, I used ImageField for avatar_url, this is where 'Pillow' package required.